packages/vscode-extension/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 4.1.3 - Nov 17, 2022
+
+Enhancements:
+- Bump up version of [@microsoft/teamsfx](https://www.npmjs.com/package/@microsoft/teamsfx) used in templates and samples to `2.0.0` that depends on [@microsoft/teams-js](https://www.npmjs.com/package/@microsoft/teams-js) version `2.0.0` and above. Starting from this version, it enables [Teams app to run in Outlook and Office.](https://learn.microsoft.com/microsoftteams/platform/m365-apps/overview)
```

Comment:
Templates are already updated in last release, and it seems that only samples are updated?

Reply:
We didn't mention updating templates from last release, so would be good to say it here.